ajax/libs/caver-js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "caver-js",
+  "title": "JavaScript API Library for Klaytn",
+  "description": "caver-js is a JavaScript API library that allows developers to interact with a Klaytn node",
+  "author": {
+    "name": "Klaytn Team",
+    "url": "https://www.klaytn.com"
+  },
+  "license": "LGPL",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/klaytn/caver-js.git"
+  },
+  "homepage": "https://github.com/klaytn/caver-js",
+  "npmName": "caver-js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
`dist` directory is not present in npm package.

Reply:
Sorry, but the changes to add the dist folder to the npm package are now only released in the rc version.
I will request a review again after it is released in latest version!

Reply:
the bot will publish rc so it should be fine

Reply:
@xtuc 
Are you saying it will be on cdn for the rc version? 
I didn't know that! Thank you for letting me know :)

Reply:
Yes